backend/dal/Helpers/Extensions/ProjectExtensions.cs

```diff
@@ -293,7 +293,7 @@ public static void DisposeProjectProperties(this PimsContext context, Entity.Pro
             var disposed = context.PropertyClassifications.AsNoTracking().FirstOrDefault(c => c.Name == "Disposed") ?? throw new KeyNotFoundException("Classification 'Disposed' not found.");
             var parentParcels = GetSubdivisionParentParcels(project);
             DisposeSubdivisionParentParcels(context, parentParcels);
-            project.Properties.ForEach(p =>
+            project.Properties.Where(p => !parentParcels.Any(pp => pp.Id == p.Id)).ForEach(p =>
```

Comment:
this prevents the parent parcels from being set to disposed (should be set to subdivided).